src/signals/IncidentMap/components/utils/update-filter-categories.ts

```diff
@@ -0,0 +1,17 @@
+import type { Filter } from '../FilterPanel'
+
+export const updateFilterCategories = (
+  categoryName: string,
+  categories: Filter[]
+) => {
+  return categories.map((category) => {
+    if (category.name === categoryName) {
+      return {
+        ...category,
+        filterActive: !category.filterActive,
+      }
+    } else {
```

Comment:
believe you can get rid of the else here

Reply:
true!